tests/ui/issues/issue-zst-nthback-148682.rs

```diff
@@ -0,0 +1,9 @@
+// @ check-pass
+//test Intolter::nth_back does not cause UB for ZSTs with high alignment
```

Comment:
Please remove leading spaces from commands, and add a space before comments.

```suggestion
//@ check-pass
// test Intolter::nth_back does not cause UB for ZSTs with high alignment
```